_sass/layout/_search.scss

```
@@ -0,0 +1,23 @@
+.search__wrapper {
+  margin-top: 20px;
+  margin-bottom: 50px;
+}
+
```

Adding this class in this file replaces the media query styles that were added in `_grid.scss`. You'll find the use of this class below in my suggestion in `search-query.jsx`, along with the other `grid` classes. It simply adds a margin below the election checkboxes on small screens.

```suggestion
.election-checkboxes {
  @include media($grid--small) {
    margin-bottom: 2 * $spacing-base;
  }
}
```